.travis.yml

```diff
@@ -11,13 +11,16 @@ env:
   - DB=mysql
   - DB=postgresql
 
+services:
+  - mongodb
+
 script:
   - RAILS_ENV=test bundle exec rake --trace db:migrate
-  - RAILS_ENV=test CODECLIMATE_REPO_TOKEN=44d7688de8e1b567b4af25ec5083c2cc0a355ab911192a7cbefd1ea25b2ffd3d bundle exec rake
+  - RAILS_ENV=test bundle exec rake
```

Comment:
did you mean to remove the `CODE_REPO_TOKEN` env variable assignment? I believe we need it for the build badge.